examples/robot_localization_failure/config.yaml

```diff
@@ -1,9 +1,16 @@
 rosbag:
-  path: ???
+  path: recordings/test_folder
```

Comment:
I'd suggest specifying a list of rosbag diretories to make the experiment more reproducible